### ISSUE TITLE
You can select multiple responses but only one is plotted. It should restrict to only one selected. todo

### ISSUE THREAD

**Issue by oysteoh** [equinor/webviz-ert#254]

**Comment by DanSava** [comment]
This is not really a bug, there is a point to being able to select more than one response in `Response correlation viewer` page. 

Once everything is selected clicking on the resulting heatmap will change the parameter/response that is on focus.


![Screenshot from 2022-05-26 17-30-42](https://user-images.githubusercontent.com/4508053/170511085-c9b8ee0f-fdca-4ff9-9aad-1acfd6044935.png)

I still think that limiting the responses to just one is more intuitive but also the heatmap will also be one-dimensional, it is unclear to me if that is good or if we are losing something by this change.

**Comment by oysteoh** [comment]
@xjules do you have any input to this issue ? Is there a way to make this more intuitive whilst keeping the functionality?

**Comment by xjules** [comment]
@DanSava @oysteoh That is the correct behaviour; ie. clicking on the heatmap will select exactly a single response and/or single parameter. At least that was the required functionally back when we were discussing this view initially. Thus the correlation is computed response vs. parameter. Not sure what would be the point of computing the correlation between a set of responses here since you will lose the impact of parameters on the selected response.

**Comment by oysteoh** [comment]
it seems the functionality - when it is understood - is what we wanted. But users find this very confusing and they doesn't understand how things are working...  Is there a way to make this simpler / easier to understand / more userfriendly?

**Comment by xjules** [comment]
> it seems the functionality - when it is understood - is what we wanted. But users find this very confusing and they doesn't understand how things are working... Is there a way to make this simpler / easier to understand / more userfriendly?

Yes, I think that is possible. I think none of the remaining plugins got any real feedback until now (also the work on webviz-ert hasn't been really prioritized). Would you create an issue for it @oysteoh ?
